ListView-and-Adapters-HW/app/src/main/java/com/elysium/listview_and_adapters_hw/Player.java

```diff
@@ -0,0 +1,43 @@
+package com.elysium.listview_and_adapters_hw;
+
+import java.util.ArrayList;
+
+/**
+ * Created by jay on 10/18/16.
+ */
+
+public class Player {
+
+    private String mName;
+    private String mPosition;
+
+    public Player(String name, String position) {
+
+        mName = name;
+        mPosition = position;
+    }
+
+    public String getmName() {
+        return mName;
+    }
+
+    public void setName(String name) {
+        mName = name;
+    }
+
+    public String getPosition() {
+        return mPosition;
+    }
+
+    public void setPosition(String position) {
+        mPosition = position;
+
+    }
+
+    // public static void Players(ArrayList<> players) {
+
+    /** can't seem to figure out how to add players to this app... I'm really getting stuck with
+     * the work-flow. Switching back and forth seems to have me losing my train of thought a bit.
```

Comment:
This class just describes a single player. You wouldn't want to create a `List` of players here - that would be done somewhere outside this class. A class just describes how an individual object can act and what data it holds. It doesn't create and store instances of itself (with the notable exception of a singleton).